themes/10up-theme/tailwind.config.js

```diff
@@ -1,4 +1,11 @@
-const tailwindProse = require('@tailwindcss/typography');
+const colors = require('tailwindcss/colors');
+
+// Tailwind Plugins
+const typographyPlugin = require('@tailwindcss/typography');
+const { getThemeColors } = require('./includes/tailwind-helpers');
+// const formsPlugin = require('@tailwindcss/forms');
+// const aspectRatioPlugin = require('@tailwindcss/aspect-ratio');
+// const lineClampPlugin = require('@tailwindcss/line-clamp');
```

Comment:
@iansvo @Nikki-Jones @dainemawer Do we want to leave these comments here so people can easily bring them back or should we just leave them out entirely and explain it in the docs?

Reply:
Hmm, I'm undecided on this one. The only negative thing about having them in the code base is that it kinda looks like a half-baked feature, so maybe the README is better. Considering we still need to document these plugins regardless, maybe the README is better.